exercises/armstrong-numbers/example.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  validate: function (input) {
+    var digits = (input + '').split('');
```

Comment:
Similar to my comment on the ES submission, let's use `String(input)` here to make it clear what is happening.

Reply:
Done, amended into the previous commit 👍